sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/properties/AzureServiceBusJmsProperties.java

```diff
@@ -310,7 +310,7 @@ public void afterPropertiesSet() throws Exception {
             }
         }
 
-        if (null == pricingTier || !pricingTier.matches("(?i)premium|standard|basic")) {
+        if (null == pricingTier || !pricingTier.matches("(?i)premium|standard")) {
             throw new IllegalArgumentException("'spring.jms.servicebus.pricing-tier' is not valid");
```

Comment:
Should we add a specific message for the `basic` tier? it tells the user that the basic tier is valid but not supported.